src/server/seeds/development/01_products_table.js

```diff
@@ -197,7 +197,7 @@ exports.seed = function (knex) {
           stock_amount: 100,
           is_on_discount: false,
           discount_percent: 0,
-          picture: 'https://via.placeholder.com/500x500',
+          picture: 'src/client/assets/images/image_23.png',
```

Comment:
image_ 23 and image_26 are missing. But the rest are working and look so nice 👍

Reply:
Hey Balint .... I am sorry but I also said    image_26      , if you change that too than you can merge no problem. 💯